Assets/Group1/Scripts/EnemySpawner.cs

```diff
@@ -0,0 +1,37 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Events;
+
+public class EnemySpawner : MonoBehaviour
+{
+    public UnityAction StopGame;
+    [SerializeField] private Enemy _enemy;
+    [SerializeField] private int _enemyCount;
+    [SerializeField] private float _spawmRaius;
+
+    private void Start()
+    {
+        for (int i = 0; i < _enemyCount; i++)
+        {
+            Vector2 enemyPosition = Random.insideUnitCircle * _spawmRaius;
+            var enemy = Instantiate(_enemy, enemyPosition, Quaternion.identity,transform);
+            enemy.GetComponent<Enemy>().DeadEnemy += OnEnemyDie;
+        }
```

Comment:
https://trello.com/c/orS9j3Rj/6-%D0%B4%D0%B8%D0%BD%D0%B0%D0%BC%D0%B8%D1%87%D0%B5%D1%81%D0%BA%D0%BE%D0%B5-%D1%81%D0%BE%D0%B7%D0%B4%D0%B0%D0%BD%D0%B8%D0%B5-%D0%BE%D0%B1%D1%8A%D0%B5%D0%BA%D1%82%D0%BE%D0%B2